guide/the-3d-window.md

```diff
@@ -19,9 +19,9 @@ the Control key (Ctrl) together with the Alt key.
 Dragging the mouse while pressing a mouse button moves the camera of the 3D
 window.
 
-- *Camera rotation*: In the 3D window, press the left button and drag the mouse to
-select an object and rotate the viewpoint around it. If no object is selected,
-the camera rotates around the origin of the world coordinate system.
+- *Camera rotation*: In the 3D window, click on an object with the left mouse
+button and drag the mouse to rotate the viewpoint around it. If you clicked on
```

Comment:
I'm not sure about using the past tense in the if clause.
Otherwise the change is fine.